File: spark-batch/src/main/scala/com/sumo/experiments/scala/Schema.scala
package com.sumo.experiments.scala

import org.apache.spark.sql.types._

object Schema {
    val peopleSchema: StructType = new StructType().add("id", "integer", nullable = false).add("name", "string").add("age", "integer").add("numFriends", "integer")
    val skillSchema: StructType = new StructType().add("pid", "integer", nullable = false).add("skill", "string").add("level", "integer")
}

File: shared/src/main/kotlin/com/sumo/experiments/Schema.kt
package com.sumo.experiments

import org.apache.spark.sql.types.*

object Schema {
    val peopleSchema = StructType().add("id", "integer", false).add("name", "string").add("age", "integer").add("numFriends", "integer")
    val skillSchema = StructType().add("pid", "integer", false).add("skill", "string").add("level", "integer")
}
